yarn.lock

```diff
@@ -6206,7 +6197,7 @@ prosemirror-collab@^1.3.1:
   dependencies:
     prosemirror-state "^1.0.0"
 
-prosemirror-commands@^1.0.0, prosemirror-commands@^1.5.0:
+prosemirror-commands@1.5.2, prosemirror-commands@^1.0.0, prosemirror-commands@^1.5.0:
```

Comment:
seems like we got a lot of duplicated dependencies with this update

Reply:
It's fine for this line to change as long as only a single package is resolved. I updated the versions in style-guide, so no new prosemirror dependencies should be pulled here.